src/components/shared/routers/AppRouter.js

```diff
@@ -82,7 +83,7 @@ class AppRouter extends React.Component {
             path="/test"
```

Comment:
should the path be /test or was it meant to be on one of our acutal pages?
it would make sense to let it be on /test if we decide to let this feature only be there for the presentation on monday and further develop it in milestone 4 sprint

Reply:
- Was for testing, thats why i left it on /test.
- Should be reusable component for other pages :D